shared/chat/manage-channels/container.js

```diff
@@ -88,6 +94,8 @@ export default compose(
       })),
     onSaveSubscriptions: props => () =>
       props._saveSubscriptions(props.oldChannelState, props.nextChannelState),
+    onPreview: ({previousPath, onPreview}) => (conversationIDKey: string) =>
+      onPreview(conversationIDKey, previousPath),
```

Comment:
I think you should call the one you made in `mapDispatchToProps` `_onPreview` otherwise it's kinda confusing.